app/controllers/todos_controller.rb

```diff
@@ -0,0 +1,5 @@
+class TodosController < ApplicationController
+	def index
```

Comment:
Thanks, I will look into updating my preferences and let you know if I need help!